templates/cf.yml

```diff
@@ -631,6 +654,9 @@ properties:
     blacklisted_syslog_ranges: ~
     unmarshaller_count: (( merge || 5 ))
     port: (( merge || 4443 ))
+    etcd:
+      client_cert: ~
+      client_key: ~
```

Comment:
@Amit-PivotalLabs Sounds good. We can use the same certs and tie them to the loggregator components that need them.